bindings_ffi/src/mls.rs

```diff
@@ -203,17 +203,22 @@ impl FfiSignatureRequest {
     pub async fn add_scw_signature(
         &self,
         signature_bytes: Vec<u8>,
-        address: String,
+        account_address: String,
         chain_id: u64,
         block_number: u64,
+        hash: Vec<u8>,
```

Comment:
The hash should not be passed in here. It can come from hashing the inner signature text for security.

Reply:
Got it